app/assets/javascripts/activeadmin_addons/inputs/select2.js

```diff
@@ -49,4 +49,7 @@ var initializer = function() {
 };
 
 $(initializer);
+$(document).on('turbolinks:before-visit',function(){
```

Comment:
A space is required after ','                    comma-spacing<br>Missing space before opening brace               space-before-blocks